sources/packages/api/src/route-controllers/application/models/application.model.ts

```diff
@@ -103,3 +106,46 @@ export interface ApplicationWithProgramYearDto {
   programYearId: number;
   active: boolean;
 }
+
+export interface SuccessWaitingStatus {
+  success: boolean;
+  waiting: boolean;
+}
+
+export interface ApplicationIncomeVerification {
+  parent1IncomeVerificationStatus: SuccessWaitingStatus;
+  parent2IncomeVerificationStatus: SuccessWaitingStatus;
+  partnerIncomeVerificationStatus: SuccessWaitingStatus;
+  studentIncomeVerificationStatus: SuccessWaitingStatus;
+}
+
+export interface ApplicationSupportingUserDetails {
+  parent1Info: SuccessWaitingStatus;
+  parent2Info: SuccessWaitingStatus;
+  partnerInfo: SuccessWaitingStatus;
+}
+
+export interface InProgressApplicationDetailsAPIOutDTO
+  extends ApplicationIncomeVerification,
+    ApplicationSupportingUserDetails {
+  id: number;
+  applicationStatus: ApplicationStatus;
+  pirStatus: ProgramInfoStatus;
+  PIRDeniedReason: string;
+  offeringStatus: OfferingStatus;
+  exceptionStatus: ApplicationExceptionStatus;
+}
+
+export interface CancelledApplicationDetailsAPIOutDTO {
+  statusUpdatedOn: Date;
+}
+
+export interface ApplicationDetailsAPIOutDTO {
+  applicationNumber: string;
+  applicationInstitutionName: string;
```

Comment:
For an object named `ApplicationDetails`, do we need to define all properties with the `application` prefix?